change: [DI-24111] - Updated hovering color for dashboard icons

nikhagra-akamai opened the pull request:
## Description 📝

Added blue color to icon on hover in dashboards

## Changes  🔄

List any change(s) relevant to the reviewer.

1. Updated global filters file to add blue color on hover
2. Updated zoomer.tsx file to add blue color on hover

## Target release date 🗓️

Please specify a release date (and environment, if applicable) to guarantee timely review of this PR. If exact date is not known, please approximate and update it as needed.

## Preview 📷

**Include a screenshot or screen recording of the change.**

:lock: Use the [Mask Sensitive Data](https://cloud.linode.com/profile/settings) setting for security.

:bulb: Use `<video src="" />` tag when including recordings in table.

| Before  | After   |
| ------- | ------- |
|![Screenshot 2025-03-19 at 11 32 01 AM](https://github.com/user-attachments/assets/e36eb9ff-6a77-4549-a00c-d0d3632e3560)|![Screenshot 2025-03-19 at 12 20 18 PM](https://github.com/user-attachments/assets/afefc316-87da-460f-990c-68a3a3299a4e)|
|![Screenshot 2025-03-19 at 11 32 09 AM](https://github.com/user-attachments/assets/605c5095-854c-47d1-adfb-4b330a72dcda)|![Screenshot 2025-03-19 at 12 20 33 PM](https://github.com/user-attachments/assets/2419f1df-86b9-4a3e-bc12-548e19698a97)|
|![Screenshot 2025-03-19 at 11 32 17 AM](https://github.com/user-attachments/assets/b07a0f89-9910-4e5b-bbda-2cfbff6f50de)|![Screenshot 2025-03-19 at 12 20 24 PM](https://github.com/user-attachments/assets/b28d20d6-347f-4c51-8839-ed8fc9d097ea)|


## How to test 🧪
1. Switch as mock user
2. Go to metrics from mega menu
3. select dashboard & required filters
4. hover on refresh and maximize/minimize button to check its changing color

<details>
<summary> Author Checklists </summary>

## As an Author, to speed up the review process, I considered 🤔

👀 Doing a self review
❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
🤏 Splitting feature into small PRs
➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
🧪 Providing/improving test coverage
 🔐 Removing all sensitive information from the code and PR description
🚩 Using a feature flag to protect the release
👣 Providing comprehensive reproduction steps
📑 Providing or updating our documentation
🕛 Scheduling a pair reviewing session
📱 Providing mobile support
♿  Providing accessibility support

<br/>

- [ ] I have read and considered all applicable items listed above.

## As an Author, before moving this PR from Draft to Open, I confirmed ✅

- [ ] All unit tests are passing
- [ ] TypeScript compilation succeeded without errors
- [ ] Code passes all linting rules

</details>

bnussman-akamai reviewed (changes requested):
Our `<IconButton />`'s default styles are to turn blue on hover. We shouldn't need to apply custom styles to accomplish this.

Also, the Icons are completely broken in dark mode

https://github.com/user-attachments/assets/33c93dd2-7479-4b75-824e-1897345ee6e3

I recommend the following
- Open up `zoomin.svg`,  `zoomout.svg`, `refresh.svg`. Change all `fill`s and `stroke`s to use `currentColor` rather than their hardcoded colors
- Remove styled icons and one-off styles completely


These changes will make the code much cleaner, simple, and has the added benefits of pulling styles from our theme. The more one-off styles we have, the harder it will be to make a cohesive application with a good user experience.

nikhagra-akamai commented:
thanks of code cleanup suggestions @bnussman-akamai I've modified the changes accordingly. I've added color prop in icons button as default button color is light but as per mockups it is a bit darker.
